### PR TITLE
added solved for DNS01 challenges

### PR DESCRIPTION
**Pull request by sidler-mozilla**

### CODE REVIEW

File: charts/refractr/templates/issuer.yaml [hunk 1]
```diff
@@ -18,4 +18,20 @@ spec:
       - http01:
           ingress:
             class: nginx
+      - dns01:
+          # This is so we can instruct LE to follow a CNAME on the target domain to
+          # a domain we control (refractr.mozit.cloud) in route53
+          cnameStrategy: Follow
+          route53:
+            region: us-west-2
+            hostedZoneID: Z06469063HC60S7SHF706
```
Comment (limed) [inline charts/refractr/templates/issuer.yaml:27]: Why hardcode this value here when you can templatize it? You should make this into a template

File: charts/refractr/templates/issuer.yaml [hunk 2]
```diff
@@ -18,4 +18,20 @@ spec:
...
+            role: "arn:aws:iam:::role/itse-apps-stage-1-cert-manager-role"
```
Comment (limed) [inline charts/refractr/templates/issuer.yaml:28]: Does the role need to live in the manifest? Shouldn't cert-manager itself just have that permission? Wouldn't annotating the service account itself with the role be more than enough.

See https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/ 

If annotating the service account doesn't work you should templatize it

File: charts/refractr/templates/issuer.yaml [hunk 3]
```diff
@@ -18,4 +18,20 @@ spec:
...
+        selector:
+          dnsNames:
```
Comment (limed) [inline charts/refractr/templates/issuer.yaml:30]: Make this into template variables

**Review by limed — CHANGES_REQUESTED** [review]
See review https://github.com/mozilla-it/helm-charts/pull/73#pullrequestreview-514287580

**Comment by limed** [comment]
Another thing I want to note which I guess goes into the whole team of "make these into template values", this PR basically ties stage and prod together since it will end up going to both environments (we don't want that).

**Review by duallain — APPROVED** [review]

**Review by limed — APPROVED** [review]
r+

**Pull request merged**